darwin/dataset/remote_dataset.py

```diff
@@ -441,13 +441,13 @@ def create_annotation_class(self, name: str, type: str, subtypes: List[str] = []
             error_handlers=[name_taken, validation_error],
         )
 
-    def add_annotation_class(self, annotation_class: AnnotationClass) -> Union[Dict, None]:
+    def add_annotation_class(self, annotation_class: Union[AnnotationClass, int]) -> Union[Dict, None]:
         """
         Adds an annotation class to this dataset.
 
         Parameters
         ----------
-        annotation_class : AnnotationClass
+        annotation_class : AnnotationClass or annotation_class_id (int)
```

Comment:
Not sure if this is the right syntax, this should be a type. The description belongs to the line below. @Fl4m3Ph03n1x could you confirm?

Reply:
The typing is correct!

Reply:
What I meant is: shouldn't those lines be
```
        Parameters
        ----------
        annotation_class : Union[AnnotationClass, int]
            The annotation class object or ID to add.
```